apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -679,12 +679,13 @@ export class DotEditContentHtmlService {
             },
             'add-contentlet': (dotAssetData: DotAssetPayload) => {
                 this.renderAddedContentlet(dotAssetData.contentlet, true)
-                    .pipe(take(1))
                     .subscribe((shouldSave: boolean) => {
                         if (shouldSave && dotAssetData.contentlet.baseType === 'FORM') {
                             this.iframeActions$.next({
                                 name: 'save'
                             });
```

Comment:
We should move this inside `renderAddedContentlet` and don't return anything to avoid unnecessary empty subscribe, which is a bad practice.